apps/fluent-tester/src/TestComponents/Avatar/InitialsAvatar.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
Will it be more examples in this file? Because this color logic could be added to CustomizedAvatar section